jcl/j9modules.xml

```diff
@@ -104,6 +125,22 @@
 			</replaceregexp>
 		</sequential>
 	</macrodef>
+	
+	<macrodef name="removeopens">
+		<attribute name="package" />
+		<sequential>
+			<echo message="Removing opening of package @{package}" />
+			<loadresource property="regexp.@{package}">
+				<string value="@{package}" />
+				<filterchain>
+					<replaceregex pattern="\." replace="\\\\." flags="g" />
+				</filterchain>
+			</loadresource>
+			<replaceregexp file="${module.dir}/classes/module-info.java" replace="" flags="gs">
+				<regexp pattern="\s*\bopens\s+${regexp.@{package}}\b[^;]*;" />
+			</replaceregexp>
+		</sequential>
+	</macrodef>
```

Comment:
Can `removeexport` and `removeopens` be generalized to a single `macrodef`?

Reply:
They can't be combined since there are modules that both export and open a package. In the case of this PR, `java.management` exports `java.lang.management` and opens it to `jdk.management`.